### PR TITLE
Updates add survey response menu

### PR DESCRIPTION
**Pull request by ryanchristo**
This pull request makes the following changes:
- Creates `mainsheet-service` and `mainsheet-container` to support new mainsheet pattern
- Creates `add-post-survey-list` directive and template so that the survey list can be displayed within the new mainsheet pattern

Testing checklist:
- [x] Create new mainsheet service and container
- [x] Create new survey list using mainsheet pattern

Fixes ushahidi/platform-client#696.

Ping @ushahidi/platform

### CODE REVIEW

**Comment by Angamanga** [comment]
I found another thing, maybe thats what you are working with right now @ryanchristo the background is supposed to fade out and become dark when the modal comes up. I'll see if I can figure it out...

File: app/common/directives/mainsheet-container.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="mainsheet" ng-show="showMainsheet" style="display: block;">
```
Comment (Angamanga) [inline app/common/directives/mainsheet-container.html]: add the classes ```init active``` after `mainsheet`. And ```opacity: 1;``` to style... And also change to ng-style if possible, uglify often mess up the 'normal' inline-styles.

File: app/main/posts/views/add-post-survey-list.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<ul>
+    <li ng-repeat="form in forms">
+        <a ng-click="handleClick(form)">
+            <span class="post-band" ng-style="{backgroundColor: form.color}"></span>
+            <svg class="iconic">
```
Comment (ryanchristo) [inline app/main/posts/views/add-post-survey-list.html]: Icon removed. Thanks!

File: app/common/directives/mainsheet-container.html (hunk 1 above)
Comment (ryanchristo) [inline app/common/directives/mainsheet-container.html]: Added `init active` and moved `opacity: 1` from `mainsheet-overlay` to `mainsheet`.

**Comment by ryanchristo** [comment]
@Angamanga Thank you! I made the requested changes. I'm not entirely sure on the `@todo` but would be happy to make any further adjustments if need be.

**Pull request merged**